src/runtime/meter-flow.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2021 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ActivityIframeView} from '../ui/activity-iframe-view';
+import {SubscriptionFlows} from '../api/subscriptions';
+import {ViewSubscriptionsResponse} from '../proto/api_messages';
+import {feUrl} from './services';
+
+export class MeterFlow {
```

Comment:
Is this not meant to share similar functionality with `MeterToastApi`? Is it possible to re-use that and just pass some different arguments (i.e. primarily the url)?

Reply:
MeterToastApi is reused.